docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,442 @@
+---
+id: molecular-data-prep
+title: Preparing and Validating Molecular Data
+platform_key: DOCS_PREPARING_MOLECULAR_DATA
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The DCC requires that metadata (necessary information describing molecular data files) is submitted at the same time as the molecular data to facilitate automated downstream analysis. This metadata is submitted to the ARGO metadata repository, called Song.  To prepare the molecular metadata, you will need to follow these steps: 
+
+1. Prepare metadata payloads according to the correct schema.
+2. Validate the payloads and molecular files with the ARGO `seq-tools` client. 
+3. Once the metadata has been prepared, both the payload and the molecular data files can be [submitted](/docs/submission/submitting-molecular-data) together. 
+
+## Preparing Metadata Payloads
+
+### Understanding the Payload Format
+
+Molecular data is submitted in conjunction with metadata submitted in JSON format, called a payload. The payload is validated against a JSON Schema to ensure data quality. The first step of submitting molecular data is to prepare the metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. Once the metadata has been prepared, both the payload and the molecular data files can be [submitted](/docs/submission/submitting-molecular-data) together.
```

Comment:
JSON Schema - don't capitalize schema like you did below

As discussed in slack...maybe say
`The first step of submitting molecular data is to prepare the metadata payloads using the examples below as guides.`
or
`The payload is validated against a JSON Schema to ensure data quality. The first step of submitting molecular data is to prepare the metadata payloads conforming to the most recent JSON schema that has been defined by the DCC. The JSON schema is complicated to follow, so we suggest using the following examples as a starting point. `

Reply:
other way around- it should be capitalized. that has been fixed. 

ive adjusted this area with the general feedbak and an idea to be more clear on examples v schema